clients/client-python/gravitino/client/gravitino_client_config.py

```diff
@@ -0,0 +1,130 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import TypeVar
+
+from gravitino.constants.timeout import TIMEOUT
+
+T = TypeVar("T")
+
+
+class GravitinoClientConfig:
+    """
+    Configuration class for Gravitino Python client;
+    It encapsulates HTTP connection configurations and validates client properties.
+    """
+
+    POSITIVE_NUMBER_ERROR_MSG: str = "The value must be a positive number"
+    """Error message for positive number validation"""
+
+    GRAVITINO_CLIENT_CONFIG_PREFIX: str = "client_"
+    """Configuration key prefix for Gravitino client config"""
+
+    CLIENT_TIMEOUT_DEFAULT: int = TIMEOUT
+    """Default HTTP request timeout in seconds"""
+
+    CLIENT_TIMEOUT: str = "client_timeout"
+    """Configuration key for request timeout"""
```

Comment:
It is better change all the client_timeout to client_request_time, not make it more clear for uers.